### PR TITLE
Dh add form hint

### PR DESCRIPTION
**Pull request by DilwoarH**
Add option hint for selection buttons 
Ref: https://govuk-elements.herokuapp.com/form-elements/#form-hint-text

screen shot 2018-06-18 at 14 31 45

### CODE REVIEW

**Comment by DilwoarH** [comment]
This one is the default from https://govuk-elements.herokuapp.com/form-elements/#form-hint-text

The hint class is used for "Reveal Hint" which comes with additional padding and custom styling

@samuelhwilliams

File: toolkit/templates/forms/_selection-button.html [hunk 1]
```diff
@@ -20,6 +20,11 @@
         {{ option.description }}
       </p>
     {% endif %}
+    {% if option.hint %}
+      <span class="form-hint">
```
Comment (galund) [inline toolkit/templates/forms/_selection-button.html]: FWIW this looks wrong to me - I think it should have been a `div`? (it's at the same level as a `p` which is block-level). Sorry I realise I'm too late and you'd have to refactor the tests on your other PR :(

Comment (DilwoarH) [reply toolkit/templates/forms/_selection-button.html]: Hey @galund , 

I personally think block-level elements should not sit inside labels. It should always be a `span`.

I found this on the internet which seems to back up what I'm saying:
https://stackoverflow.com/a/4461959

The `p` tag above should probably change to a `span`

Comment (galund) [reply toolkit/templates/forms/_selection-button.html]: Oh yeah, see what you mean. 👍

**Pull request merged**